src/honey/sql.cljc

```diff
@@ -214,16 +215,16 @@
      (fn [fk _] (param-value (fk)))}))
 
 (defn- format-var [x & [opts]]
-  (let [c (name-_ x)]
+  (let [c (name x)]
     (cond (= \% (first c))
-          (let [[f & args] (str/split (subs c 1) #"\.")]
-            ;; TODO: this does not quote arguments -- does that matter?
-            [(str (upper-case f) "(" (str/join "," args) ")")])
+          (let [[f & args] (str/split (subs c 1) #"\.")
+                quoted-args (map #(format-entity (keyword %) opts) args)]
+            [(str (upper-case (-_ f)) "(" (str/join ", " quoted-args) ")")])
```

Comment:
Instead of introducing a new function (see comment above), you can use `(sql-kw f)` as I suggested, instead of `(upper-case (-_ f))`, and that will be more in keeping with similar code elsewhere.